src/main/java/uk/ac/ebi/embl/api/validation/dao/EraproDAOUtilsImpl.java

```diff
@@ -753,48 +672,34 @@ private void setXrefs(String refs, Entry masterEntry) {
 		}
 	}
 
-	private SampleInfo getSampleInfoFromBiosamplesSample(Sample biosamplesSample) {
+	private SampleInfo getSampleInfo(Sample sample) {
```

Comment:
- I suggest removing SampleInfo class and using the Sample class instead. Looking at the code this could be very quick change.
- - I suggest removing SampleEntity class and using the Sample class instead. Looking at the code this could be very quick change.